server/utils.go

```diff
@@ -21,9 +16,12 @@ func (s *Server) logToMattermost(msg string, args ...interface{}) {
 
 	webhookRequest := &Payload{Username: "Mattermod", Text: webhookMessage}
 
-	if err := s.sendToWebhook(webhookRequest); err != nil {
+	r, err := s.sendToWebhook(s.Config.MattermostWebhookURL, webhookRequest)
+	if err != nil {
 		mlog.Error("Unable to post to Mattermost webhook", mlog.Err(err))
+		return
 	}
+	defer r.Body.Close()
```

Comment:
Same comment as earlier.

Reply:
I am not seeing this resolved. Please see my comment https://github.com/mattermost/mattermost-mattermod/pull/143/#discussion_r446847856. We can just use `closeBody` here.